config/v1/types_infrastructure.go

```diff
@@ -675,6 +749,18 @@ type GCPPlatformStatus struct {
 	// +optional
 	// +nullable
 	CloudLoadBalancerConfig *CloudLoadBalancerConfig `json:"cloudLoadBalancerConfig,omitempty"`
+
+	// serviceEndpoints specifies endpoints that override the default endpoints
+	// used when creating clients to interact with GCP services.
+	// serviceEndpoints is optional.
```

Comment:
When not specified, the behaviour is to use the default endpoints for the region that the cluster is deployed to? Is that correct?

Reply:
yes. The setup would end up looking something like https://github.com/barbacbd/k8s-cluster-api-provider-gcp/blob/main/cloud/scope/clients.go#L153. That way the option is just not set when it is not present.

Reply:
So would it be fair to say
```suggestion
	// When not specified, the default endpoints for the GCP region will be used.
```